README.md

```diff
@@ -51,30 +44,18 @@ and [`http-message-implementation`](https://packagist.org/providers/psr/http-mes
 ## Usage
 
 ```php
-namespace XXX;
-
-use Sentry\ClientBuilder;
-
-require 'vendor/autoload.php';
+use function Sentry\init;
+use function Sentry\captureException;
 
-// Instantiate the SDK with your DSN
-$client = ClientBuilder::create(['server' => 'http://public@example.com/1'])->getClient();
+init(['dsn' => '___PUBLIC_DSN___' ]);
```

Comment:
Should we also add an example on how to register the error handler for automatic error reporting?

Reply:
Isn't it registered automatically when using the default integrations?

Reply:
Right